bidict/_orderedbase.py

```diff
@@ -98,7 +98,7 @@ def new_last_node(self) -> Node:
         """Create and return a new terminal node."""
         old_last = self.prv
         new_last = Node(old_last, self)
-        old_last.nxt = self.prv = new_last
+        old_last.nxt = self.prv = new_last  # pyright: ignore  # https://github.com/microsoft/pyright/issues/3172
```

Comment:
This bug appears to not be fully fixed in pyright 1.1.229.

Without this suppression comment, pyright 1.1.229 still flags this line with:
```console
  /Users/jab/src/bidict/bidict/_orderedbase.py:101:29 - error: "prv" is not specified in __slots__ (reportGeneralTypeIssues)
```

I left a comment on the associated issue.

In the meantime, great to now be able to use `pyright: ignore` here. Really feels like the best UX for users in this situation.

Reply:
(hit "enter" too soon)
Big thanks again to @erictraut for staying open to that suggestion. I hope it turns out to stand the test of time as more users adopt pyright.

Reply:
(For completeness, the correct link to the issue tracking this bug is https://github.com/microsoft/pyright/issues/3183)